### DISCUSSION TITLE
Delete Session

### DISCUSSION THREAD

**Discussion by rohan-gt** [langfuse/langfuse#1896] [Ideas]
### Describe the feature or potential improvement

Is it possible to add a button to delete sessions similar to how traces are deleted? The sessions tab gets very cluttered otherwise even after deleting the underlying traces.

**Comment by Leo310** [comment]
Furthermore, on session delete, it would be great to add the option to automatically also delete all traces that belong to that session.

**Comment by eduardoret4** [comment]
please implement this, it would be very useful
